### ISSUE TITLE
Image and mask output have wrong type

### ISSUE THREAD

**Issue by cardosofelipe** [AuroBit/ComfyUI-OOTDiffusion#18]
The output of image and mask  are a tuple of lists, they should be tuple of tensors in order to be compatible with other components.
To fix this in __init__.py just change the output lineof generate method to:
return (torch.stack([output_image]), torch.stack([masked_vton_img]))

**Comment by iyume** [comment]
Thanks to your advice, I have changed this.

Meantime, this project doesn't support batch evaluation yet. (Actually most of ComfyUI node so)
